src/sgame/sg_bot_skilltree.cpp

```diff
@@ -24,6 +24,44 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
 
 #include "sg_bot_util.h"
+#include "../shared/parse.h"
+
+// aliens have 61 points to spend max
+// humans have 40 points to spend max
+Cvar::Cvar<int> g_skillsetBudgetAliens( "g_skillsetBudgetAliens", "the skillset budget for aliens.", Cvar::NONE, 61 );
+Cvar::Cvar<int> g_skillsetBudgetHumans( "g_skillsetBudgetHumans", "the skillset budget for humans.", Cvar::NONE, 40 );
+
+static std::set<std::string> bg_allowedSkillset;
+
+static std::set<std::string> BG_ParseAllowedSkillsetList( const std::string &behaviorsCsv )
+{
+	std::set<std::string> skills;
+
+	for (Parse_WordListSplitter i(behaviorsCsv); *i; ++i)
+	{
+		skills.insert( *i );
+	}
+
+	return skills;
+}
+
+static void BG_SetAllowedSkillset( Str::StringRef behaviorsCsv )
+{
+	bg_allowedSkillset = BG_ParseAllowedSkillsetList( behaviorsCsv );
+}
+
+static bool BG_SkillAllowed( Str::StringRef behavior )
+{
+	return bg_allowedSkillset.find( behavior ) != bg_allowedSkillset.end();
+}
+
+static Cvar::Callback<Cvar::Cvar<std::string>> g_allowedSkillset(
```

Comment:
Might it be better to do disabled skill set instead of allowed? This also means that we don't have to keep this in sync if we add new ones. Also if we decide that allowed skillset is better, let's at least generate this list dynamically.

Reply:
It is either a blacklist or a whitelist. After some discussion on IRC, I went for the whitelist. Creating the list dynamically would be nice, but it intimately tied to the two magic numbers right now.

Reply:
I can guarantee that this will be the reason why someone spends hours trying to figure out why their skill isn't working 😄 

I'm a vote in favor of a denylist but if others feel otherwise, I'll defer.

Reply:
I have no strong opinion on this.

Reply:
I changed to a a denylist. There are already several `g_disabled*` cvars, so this should follow this convention.